motors/rcx_motor_defs.c

```diff
@@ -23,6 +23,69 @@
  */
 
 const struct rcx_motor_info rcx_motor_defs[] = {
+		[LEGO_9V_MICROMOTOR] = {
+		/**
+		 * @vendor_name: LEGO
+		 * @vendor_part_number: 2986
```

Comment:
I just saw that http://philohome.com/motors/motorcomp.htm has the "official" LEGO part number for many of these motors too. That is the preferred number here. So, this one would be `70823` instead of `2986`. Same goes for others below.